docker/server-jetty/src/main/server-jetty/requirements.txt

```diff
@@ -1,19 +1,20 @@
-adbc-driver-manager==1.1.0
-adbc-driver-postgresql==1.1.0
+adbc-driver-manager==1.2.0
+adbc-driver-postgresql==1.2.0
 connectorx==0.3.3; platform.machine == 'x86_64'
 deephaven-plugin==0.6.0
-importlib_resources==6.4.3
+docstring_parser>=0.16
```

Comment:
*Almost*; this file is a "freeze", so it should include the exact dependencies. Sorry this stuff is not all automated...